glance/files/kilo/glance-api.conf.Debian

```diff
@@ -304,20 +304,25 @@ scrubber_datadir = /var/lib/glance/scrubber
 
 # The maximum number of image members allowed per image
 #image_member_quota = 128
+{% if server.quota is defined %}{% if server.quota.image_member_quota is defined %}image_member_quota = {{ server.quota.image_member_quota|default('128') }}{% endif %}{% endif %}
```

Comment:
Put conditionals on separate lines to be more readable and use `{%-` to get rid of empty lines